finetuning/generalists/training/electron_microscopy/mito_nuc/obtain_mito_nuc_em_datasets.py

```diff
@@ -0,0 +1,167 @@
+import os
+import numpy as np
+from math import ceil, floor
+
+from elf.io import open_file
+
+from torch_em import get_data_loader
+from torch_em.transform.label import PerObjectDistanceTransform
+from torch_em.data import ConcatDataset, MinInstanceSampler, datasets
+
+from micro_sam.training import identity
+
+
+class ResizeRawTrafo:
```

Comment:
The trafos should be refactored, see comments in #292.

Reply:
These have been taken care of now. Thanks!